### ISSUE TITLE
[BUG]  - likely actually a feature request, but to the user it feels likea bug because of lack of docs 

### ISSUE THREAD

**Issue by reliablecontrols** [microsoft/playwright#27686]
tl;dr.

First off. Excellent product, my feedback is not a criticism of the product or the developers. Just sharing my user experience and how a small improvement would make my work SOOO much easier.

**Context:**
- Playwright for VSCode extension version: [e.g. 1.2.3]
- Last updated
2023-10-5, 01:59:25
Identifier
ms-playwright.playwright

- Browser: [e.g. All, Chromium, Firefox, WebKit]
- chromium
==================
issue:
In the extension when I access 
Testing->Test Explorer->Filter 

I see that it includes the text "Filter(e.g. text, !exclude, @tag).

This seems to imply that I can query for my testcases by tags and that the usage would be @<mytagname> but this is not the case at all. The text in the UI seems to imply that I would be able to use the actual @mytagname to limit my search to a particular tag or group of tags,

If I type the "@" symbol, I find every testcase that contains @ in the testcase name.  

What I actually would like to do would be to filter by @tag1 or @tag2 in the same way it can be done from the command line. (so they can be additive, or if it is regex, I could query it that way. 

The use case:
I have testcases that expliclty test out that my DOM and its associated methods that I have encoded within the playwright framework. Then I use those methods to build up more complicated sequences of steps/actions.

Using tags, it is easy to catagorize testcases as 
1. model validators
2. model users (actual testcases that use the)

Looking at the source code as quickly as I could, it looks like this is processed as a regex. 
if that were the case, adding another options (below fuzzy match) called regex that would allow the inputed string  to be treated as a regex expression.

ie. \B@test   
would result show the tetscases with the labels  @test1 and @test2

a more likely usecase would be 
@requiredtest and @optionaltest 

according to the docs
https://playwright.dev/docs/test-annotations  using a similar format like --grep "@label1|@label2" would also work.

**Comment by reliablecontrols** [comment]
@mxschmitt , thanks for your response. I will see if I can raise the issue over there.
